python/MOLTRESXS.md

```diff
@@ -0,0 +1,50 @@
+MoltresXS
```

Comment:
Strongly recommend not using capital letters for python toolkit names (or module names... or package names). PEP8 would suggest, in this case, moltresxs or, more likely: `moltres_xs`. 
```suggestion
moltres_xs
```